networkx/algorithms/dag.py

```diff
@@ -848,6 +950,12 @@ def dag_longest_path_length(G, weight="weight", default_weight=1):
     NetworkXNotImplemented
         If `G` is not directed
 
+    Examples
+    --------
+    >>> DG = nx.DiGraph([(1, 2), (2, 3)])
```

Comment:
Same comment here - a simple graph with multiple paths might be more instructive.